### PR TITLE
NXP Sabre_iMX6SX_1GB board suport added.

### PR DESCRIPTION
**Pull request by mvarecha**

### CODE REVIEW

File: hals/halext/HalExtiMXDma/iMX6SxDmaHw.h [hunk 1]
```diff
@@ -195,7 +195,7 @@ SDMA_CHANNEL_CONFIG Imx6SxDmaReqToChannelConfig[] = {
     // (3) SDMA_REQ_HDMI_AUDIO
     {
         SDMA_UNSUPPORTED_REQUEST_ID,    // Script address
-        SDMA_REQ_IPU1,                  // DMA request ID
+        SDMA_REQ_HDMI_AUDIO,            // DMA request ID
```
Comment (hbeberman) [inline hals/halext/HalExtiMXDma/iMX6SxDmaHw.h:198]: What's the impact of this change?
If it's important we'll need to rebuild the DMA HAL extension and get it re-signed.

File: build/board/Sabre_iMX6SX_1GB/Sabre_iMX6SX_1GB_FMFileList.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FMCollectionManifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns="http://schemas.microsoft.com/embedded/2004/10/ImageUpdate" Product="IoTUAP" >
+  <IsBuildFeatureEnabled>true</IsBuildFeatureEnabled>
+  <FMs>
+    <FM ID="Sabre6Q" Path="$(FMDirectory)\Sabre_iMX6SX_1GB_DeviceFM.xml" ReleaseType="Production" OwnerName="NXP" OwnerType="OEM" CPUType="ARM" Critical="true"/>
```
Comment (hbeberman) [inline build/board/Sabre_iMX6SX_1GB/Sabre_iMX6SX_1GB_FMFileList.xml]: Looks like this is a bug in my newboard script. Its not critical, it just changes the name of some temporary files, but might as well stay consistent.
```suggestion
    <FM ID="Sabre6SX" Path="$(FMDirectory)\Sabre_iMX6SX_1GB_DeviceFM.xml" ReleaseType="Production" OwnerName="NXP" OwnerType="OEM" CPUType="ARM" Critical="true"/>
```

File: hals/halext/HalExtiMXDma/iMX6SxDmaHw.h (hunk 1 above)
Comment (christopherco) [reply hals/halext/HalExtiMXDma/iMX6SxDmaHw.h:198]: Looks like a typo in the original code. Doesn't look like this entry is used at all.

Comment (mvarecha) [reply hals/halext/HalExtiMXDma/iMX6SxDmaHw.h:198]: Channels are restricted by an ASSERT() in the code to have the same DMA request ID.

File: build/board/Sabre_iMX6SX_1GB/Sabre_iMX6SX_1GB_FMFileList.xml (hunk 2 above)
Comment (mvarecha) [reply build/board/Sabre_iMX6SX_1GB/Sabre_iMX6SX_1GB_FMFileList.xml]: Fixed.

**Review by christopherco — APPROVED** [review]

**Review by hbeberman — APPROVED** [review]

**Pull request merged**